Source/WebKit/WebProcess/WebCoreSupport/WebStorageConnection.cpp

```diff
@@ -65,14 +65,15 @@ void WebStorageConnection::getEstimate(WebCore::ClientOrigin&& origin, StorageCo
 void WebStorageConnection::fileSystemGetDirectory(WebCore::ClientOrigin&& origin, StorageConnection::GetDirectoryCallback&& completionHandler)
 {
     auto& connection = WebProcess::singleton().ensureNetworkProcessConnection().connection();
-    connection.sendWithAsyncReply(Messages::NetworkStorageManager::FileSystemGetDirectory(origin), [completionHandler = WTFMove(completionHandler)](auto result) mutable {
+    connection.sendWithAsyncReply(Messages::NetworkStorageManager::FileSystemGetDirectory(origin), [origin = WTFMove(origin), completionHandler = WTFMove(completionHandler)](auto result) mutable {
```

Comment:
moving origin while using it in the same line might cause issue.
Probably worth not moving it in the lambda.